lib/Command/Bot/Install.php

```diff
@@ -76,9 +77,9 @@ protected function configure(): void {
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$name = $input->getArgument('name');
-		$secret = $input->getArgument('secret');
 		$url = $input->getArgument('url');
 		$description = $input->getArgument('description') ?? '';
+		$secret = $input->getOption('secret') ?? bin2hex(random_bytes(32));
```

Comment:
bin2hex means it's 0-9a-f, I'd use ISecureRandom instead so it's using a-zA-Z0-9